packages/allow-scripts/src/yarn-berry-plugin.cjs

```diff
@@ -0,0 +1,22 @@
+// @ts-check
```

Comment:
We already have this as a typescript thing in the packages. Do we want to replace with a no-build option?

Reply:
I didn't want to throw a tsconfig in there